src/commands/api/deployWorkspaceProjectApi.ts

```diff
@@ -38,12 +48,14 @@ export async function deployWorkspaceProjectApi(deployWorkspaceProjectOptions: a
             rootFolder,
             srcPath: srcPath ? Uri.file(srcPath).fsPath : undefined,
             dockerfilePath: dockerfilePath ? Uri.file(dockerfilePath).fsPath : undefined,
+            newRegistryCredentialType: RegistryCredentialType.DockerLogin,
             shouldSaveDeploySettings: !!shouldSaveDeploySettings,
         });
 
         const deployWorkspaceProjectContext: DeployWorkspaceProjectContext = await deployWorkspaceProjectInternal(deployWorkspaceProjectInternalContext, {
             suppressActivity: true,
             suppressConfirmation,
+            suppressRegistryPrompt: suppressRegistryPrompt ?? true,
```

Comment:
This [PR](https://github.com/microsoft/vscode-azurecontainerapps/pull/809) introduced the prompt, so add ability to suppress and default it to true to preserve existing behavior